scripts/gen_vimdoc.py

```diff
@@ -85,17 +87,24 @@
         # file patterns used by doxygen
         'file_patterns': '*.h *.c',
         # Only function with this prefix are considered
-        'func_name_prefix': 'nvim_',
+        'fn_name_prefix': 'nvim_',
         # Section name overrides.
         'section_name': {
             'vim.c': 'Global',
         },
+        # For generated section names.
+        'section_fmt': lambda name: f'{name} Functions',
```

Comment:
I get a syntax error with flake8 and https://github.com/microsoft/python-language-server. Which python3 version does this script target?

Reply:
i've been using python 3.6, but no objection to fixing the syntax

Reply:
It's ok. I upgraded to python 3.8 via pyenv. python 3.5 seems to be EOL (https://www.python.org/dev/peps/pep-0478/).